### PR TITLE
Placement Feature (ArcGIS Map View and List View)

### PR DESCRIPTION
**Pull request by eckdanny-osi**
👀 

map

### CODE REVIEW

**Comment by SashaKyz** [comment]
Can one of the admins verify this patch?

**Comment by eckdanny-osi** [comment]
@edgibbs We're out of **WiP** mode now

**Comment by eckdanny-osi** [comment]
Some weird feedback from codeclimate...

 > Replace ··}⏎···· with }

^ this is _why_ we use `prettier` in the 1st place. No issues with the linter in jenkins or locally. Maybe codeclimate rev'd their eslint engine or something. 9 of the 11 issues are formatting like this.

The other 2 are TODOs (which are deliberate).

File: app/javascript/placement/PlacementContainer.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  it('renders', () => {
+    const wrapper = mkWrapper();
+    expect(!!wrapper).toBe(true);
+  });
+
+  describe('#fetchRelatedClients', () => {
+    it('ais a function', () => {
+      const wrapper = mkWrapper();
+      expect(wrapper.instance().fetchRelatedClients).toEqual(
```
Comment (edgibbs) [inline app/javascript/placement/PlacementContainer.test.js]: elaborate

File: app/javascript/placement/_components/placement-list/PlacementList.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('PlacementList', () => {
+  it('should pass a test', () => {
+    expect(true).toBe(true);
```
Comment (edgibbs) [inline app/javascript/placement/_components/placement-list/PlacementList.test.js]: a little more tests

**Comment by edgibbs** [comment]
Will need to explore why code climate is getting different results for linting.

**Pull request merged**